sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/MSIToken.java

```diff
@@ -35,15 +36,11 @@
      * @param token the token string.
      * @param expiresOn the expiration time.
      */
-    public MSIToken(String token, OffsetDateTime expiresOn) {
```

Comment:
Is this a Breaking change ?

Reply:
This is in the implmentation package so we should be fine